Sources/HttpPipeline/NIO.swift

```diff
@@ -1,28 +1,29 @@
 import Foundation
 import NIO
 import NIOHTTP1
+import NIOHTTPCompression
 import Optics
 import Prelude
 
 public func run(
   _ middleware: @escaping Middleware<StatusLineOpen, ResponseEnded, Prelude.Unit, Data>,
   on port: Int = 8080,
+  eventLoopGroup: EventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: System.coreCount),
```

Comment:
do we need this anymore? this was for the nio Postgres stuff right?

Reply:
I think it's good to leave in for customization.